.konflux/operator/konflux.Dockerfile

```diff
@@ -0,0 +1,41 @@
+ARG OPERATOR_VERSION
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.23 as builder
+
+WORKDIR /go/src/github.com/openshift-kni/numaresources-operator
+COPY . .
+
+ENV GOEXPERIMENT=strictfipsruntime
+ENV CGO_ENABLED=1
+ENV GOTAGS="strictfipsruntime"
+
+# Build
+RUN \
+	NRO_BUILD_VERSION=${OPERATOR_VERSION} \
+	make binary-all
+
+FROM registry.redhat.io/rhel9-4-els/rhel-minimal:9.4
+
+COPY --from=builder /go/src/github.com/openshift-kni/numaresources-operator/bin/manager /bin/numaresources-operator
+# bundle the operand, and use a backward compatible name for RTE
+COPY --from=builder /go/src/github.com/openshift-kni/numaresources-operator/bin/exporter /bin/resource-topology-exporter
+COPY --from=builder /go/src/github.com/openshift-kni/numaresources-operator/bin/buildinfo.json /usr/local/share
+
+RUN mkdir /etc/resource-topology-exporter/ && \
+    touch /etc/resource-topology-exporter/config.yaml
+RUN microdnf install -y hwdata && \
+    microdnf clean -y all
+USER 65532:65532
+ENTRYPOINT ["/bin/numaresources-operator"]
+LABEL com.redhat.component="numaresources-operator-container" \
+      name="openshift4/numaresources-operator-rhel9" \
+      version="v${OPERATOR_VERSION}" \
+      summary="numaresources-operator" \
+      io.openshift.expose-services="" \
```

Comment:
is this empty on purpose?

Reply:
yes. all labels taken exactly as they were used d/s